extensions/kubectl/command.go

```diff
@@ -96,7 +96,11 @@ func Command(client *rancher.Client, yamlContent *management.ImportClusterYamlIn
 		return "", err
 	}
 
-	steveClient := client.Steve
+	steveClient, err := client.Steve.ProxyDownstream(clusterID)
```

Comment:
steveClient := client.Steve is needed for local cluster.

can you add in a condition where if the clusterID is local we set steveClient := client.Steve. if not, steveClient, err := client.Steve.ProxyDownstream(clusterID)

Reply:
Yes, I can

Reply:
if this works with local cluster also, then we dont need to add the condition..

Reply:
Actually, it is just working on RKE2 clusters, I tested in RKE1 and K3S and it is not working the if will be necessary